phpBB/docs/events.md

```diff
@@ -854,27 +860,6 @@ posting_editor_bbcode_status_after
 * Since: 3.1.4-RC1
 * Purpose: Add content after bbcode status
 
-posting_editor_buttons_after
```

Comment:
The events are being deleted because `posting_buttons.html` file is a goner. There's really no viable alternative.